entsoe-cgmes-balances-adjustment/src/main/java/com/powsybl/entsoe/cgmes/balances_adjustment/data_exchange/DataExchangesXml.java

```diff
@@ -48,9 +49,9 @@ private static class ParsingContext {
 
         private String datasetMarketDocumentMRId;
 
-        private Interval period;
+        private Pair<ZonedDateTime, ZonedDateTime> period;
```

Comment:
Don't you prefer to keep an Interval class? Up to you, but you could also use org.threeten.extra.Interval which is used in powsybl-core (in powsybl-time-series-api module).

Reply:
Yes it will indeed be cleaner, I will check it out!